test/int.jl

```diff
@@ -305,3 +305,18 @@ let i = MyInt26779(1)
     @test_throws MethodError i << 1
     @test_throws MethodError i >>> 1
 end
+
+@testset "rounding division" begin
+    for (a, b, nearest, away, up) in (
+            (3, 2, 2, 2, 2),
+            (-3, 2, -2, -2, -1),
+            (5, 2, 2, 3, 3),
+            (-5, 2, -2, -3, -2))
```

Comment:
Add some odd divisor cases to the test cases. Also `typemin`/`typemax` divisors/dividends.